packages/cubejs-base-driver/src/BaseDriver.ts

```diff
@@ -299,6 +308,59 @@ export abstract class BaseDriver implements DriverInterface {
     }
   }
 
+  public getSchemas() {
+    const query = `
+      SELECT table_schema as ${this.quoteIdentifier('schema_name')}
+      FROM information_schema.tables
+      WHERE table_schema NOT IN (${this.getIgnoredSchemas()})
+      GROUP BY table_schema
+    `;
+    return this.query<QuerySchemasResult>(query);
+  }
+
+  public getTablesForSpecificSchemas(schemas?: QuerySchemasResult[]) {
+    if (!schemas) {
+      throw new Error('getTables without schemas is not supported');
+    }
+
+    const schemasPlaceholders = schemas.map((_, idx) => this.param(idx)).join(', ');
+    const schemaNames = schemas.map(s => s.schema_name);
+
+    const query = `
+      SELECT table_schema as ${this.quoteIdentifier('schema_name')},
+            table_name as ${this.quoteIdentifier('table_name')}
+      FROM information_schema.tables
+      WHERE table_schema IN (${schemasPlaceholders})
+    `;
+    return this.query<QueryTablesResult>(query, schemaNames);
+  }
+
+  public getColumnsForSpecificTables(tables: QueryTablesResult[]) {
+    const conditions: string[] = [];
+    const parameters: string[] = [];
+    let paramIndex = 0;
+
+    tables.forEach((t) => {
+      const schemaPlaceholder = this.param(paramIndex++);
+      const tablePlaceholder = this.param(paramIndex++);
+      conditions.push(`(table_schema=${schemaPlaceholder} AND table_name=${tablePlaceholder})`);
```

Comment:
i belive it's better to to pack all tables in schema by `(table_schema = 'myschema' AND table_name IN ('table1', 'table2', 'table3')) OR (table_schema = 'yetanotherschema' AND table_name IN ('table55', 'table555'))`